build.sbt

```diff
@@ -13,6 +13,7 @@ name := Settings.Definitions.name
 
 organization in Global := "edu.gemini.ocs"
 
+Global / onChangedBuildSource := ReloadOnSourceChanges
```

Comment:
I'm curious how this will work out. Worth a try.

Reply:
I added that to stop sbt bugging me, it seemed to work the couple of times I made a change